v2.0/src/config/config.json

```diff
@@ -1,7 +1,7 @@
 {
   "IS_LOCAL": false,
   "IS_PROD": false,
-  "IS_CANARY": false,
+  "IS_CANARY": true,
```

Comment:
You don't need to commit config.json, because I always need to check it and make it right.  
Can you just commit the other changed files but not this?